andela/travel_tool_back
# #160196856 Make request available for approval

#### What does this PR do?
This PR creates an endpoint for capturing new requests and making it available in the approvals table for manager's approval decision.

#### Description of Task to be completed?
- Create an endpoint for adding new approvals into the approvals table
-  Make the new request data available in the approvals table
- Write test for the endpoint

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Change into the project directory -` cd travel_tool_back`
- Navigate to [travela](http://travela-staging.andela.com/) in your browser and login
- Open the developer tools, locate applications tab, copy `jwt-token` in the cookies
- Check `.env.example` and set up your `.env` appropriately
- Get back to your command line, Install dependencies and start the server - `make start`
- In`Postman`, use the `jwt-token` to send a request by adding a `Header`, key `Authorization` and value the `token`.
- In Postman, enter the necessary data in the `x-www-form-urlencoded` options.
- You should be able to post new request on `localhost:5000/api/v1/requests`
- The responses should include the new created request data and approval data

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160196856](https://www.pivotaltracker.com/story/show/160196856)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2018-09-05 at 17 56 17" src="https://user-images.githubusercontent.com/12033494/45103804-6628c400-b139-11e8-9b8f-0685f906297e.png">



#### Questions:
None
